packages/core/src/router.ts

```diff
@@ -39,3 +39,29 @@ export function newRouterExecuteTransactionRequest(
 
   return { to: getContractAddress(chainId, 'Router'), data, value };
 }
+
+export interface NewRouterExecuteWithSignerFeeTransactionOptions {
+  chainId: number;
+  routerBatchLogics: IParam.LogicBatchStruct;
+  signer: string;
+  signature: BytesLike;
+  tokensReturn?: string[];
```

Comment:
```suggestion
  tokensReturn: string[];
```
Could remove `?` to skip the check for `undefined`

Reply:
I assume this is necessary to keep this to allowing `tokensReturn`, `value`, `referralCode` to be optional.